spec/controllers/events_controller_spec.rb

```diff
@@ -105,7 +105,7 @@
       let(:params) { FactoryGirl.attributes_for(:event) }
 
       it 'throws a not authorized error' do
-        expect { subject }.to raise_error(Pundit::NotAuthorizedError)
+        expect(subject).to redirect_to forbidden_path
```

Comment:
If the error is no longer being thrown in the tests, then there is no reason to test for it